pkg/runner/local_docker.go

```diff
@@ -525,7 +531,10 @@ func (r *LocalDockerRunner) Run(ctx context.Context, input *api.RunInput, ow io.
 			err := cli.ContainerStart(ctx, id, types.ContainerStartOptions{})
 			if err == nil {
 				log.Debugw("started container", "id", id)
-				started <- id
+				select {
+				case started <- id:
+				default:
```

Comment:
panic?

Reply:
I don't think so, this is to mitigate a potential race between a `ContainerStart`s that were in progress, and have _just_ returned in success, just after the context was cancelled and we closed the `started` channel.

Reply:
Basically we want to discard this container ID because it would anyway be deleted by `deleteContainers`, which deletes all _created_ containers.

Reply:
Can we select on the context instead?

Reply:
I guess if the select is on the context returned by errgroup.WithContext(), yes. I'll do that.